extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/linkHandler/YoutubeParsingHelper.java

```diff
@@ -351,12 +428,8 @@ public static String fixThumbnailUrl(String thumbnailUrl) {
         return thumbnailUrl;
     }
 
-    public static JsonArray getJsonResponse(String url, Localization localization) throws IOException, ExtractionException {
-        Map<String, List<String>> headers = new HashMap<>();
-        headers.put("X-YouTube-Client-Name", Collections.singletonList("1"));
-        headers.put("X-YouTube-Client-Version", Collections.singletonList(getClientVersion()));
-        final Response response = getDownloader().get(url, headers, localization);
-
+    public static String getValidResponseBody(final Response response)
```

Comment:
Maybe `getValidJsonResponseBody` would be better to represent what is actually done. But meh, up to you to decide.

Reply:
I agree